src/blocks/header.scss

```diff
@@ -0,0 +1,62 @@
+body {
+  font-size: 12px;
+}
+
+.header {
+  margin: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+  }
+}
+
+.nav {
```

Comment:
styles for nav BEM block should be in separate file

Reply:
you didn't fix this comment
styles for nav BEM block should be in separate css file